Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -156,13 +159,14 @@ private IMixedRealityController GetOrAddWindowsMixedRealityController(Interactio
             }
 
             var controllingHand = interactionSourceState.source.handedness == InteractionSourceHandedness.Left ? Handedness.Left : Handedness.Right;
+            IMixedRealityInputSource controllerInputSource = inputSystem?.RequestNewGenericInputSource($"Mixed Reality Controller {controllingHand}");
 
             //TODO - Controller Type Detection?
             //Define new Controller
             var detectedController = new WindowsMixedRealityController(
                 ControllerState.NotTracked,
                 controllingHand,
-                inputSystem.RequestNewGenericInputSource($"Mixed Reality Controller {controllingHand}"),
+                controllerInputSource,
                 new Dictionary<DeviceInputType, InteractionMapping>()
```

Comment:
we can just pass null here, correct?

Reply:
or nothing cause null is default

Reply:
If the RequestNewGenericInputSource returns null, then it will be null.
That was the pattern we settled on, unless you want to alter it now?

Reply:
in the constructor it will create a new dictionary buy default is all I'm saying. So you don't have to make a new one here.